v0/destinations/hs/HSTransform-v2.js

```diff
@@ -52,18 +71,43 @@ const processIdentify = async (message, destination, propertyMap) => {
   const traits = getFieldValueFromMessage(message, "traits");
   const mappedToDestination = get(message, MappedToDestinationKey);
   const operation = get(message, "context.hubspotOperation");
+  const externalIdObj = getDestinationExternalIDObject(message, "HS");
+  const { objectType } = getDestinationExternalIDInfoForRetl(message, "HS");
```

Comment:
```suggestion
  const externalIdObj = getDestinationExternalIDObjectForRetl(message, "HS");
  const { objectType } = getDestinationExternalIDInfoForRetl(message, "HS");
```

Reply:
This returns the whole object. Going forward externalId structure might be more dynamic. Didn't want to extend the existing util method for this reason. That is being used in lot of places. 

Didn't add retl since there is nothing specific to retl here. I guess externalId object format is similar for event streams and Retl

Reply:
In the event stream, we do not extract the `extIdObj` attached with the destination name this way
`type.includes(`${destination}-`)`
we just compare the type simply:
`extIdObj.type === type`

Reply:
sure, then let me update the name